kolibri/core/assets/src/api-resource.js

```diff
@@ -186,7 +186,9 @@ class Model {
   set(attributes) {
     // force IDs to always be strings - this should be changed on the server-side too
     if (attributes && this.resource.idKey in attributes) {
-      attributes[this.resource.idKey] = String(attributes[this.resource.idKey]);
+      if(attributes[this.resource.idKey]) { // but null or undefined stays the same good old.
```

Comment:
whitespace: `if (attributes[this.resource.idKey])`

also, I don't really understand the comment